server/config/routes/contacts.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { saveNewContact,
+         updateContactById,
+         getContactById,
+         updateContactDoNotCallById } from '../../controllers/contacts';
+// add updateContactInvalidNumberById
+const router = express.Router();
+
+router.route('/').post(saveNewContact);
+router.route('/:id').get(getContactById);
+router.route('/:id').put(updateContactById);
+router.route('/:id').patch(updateContactDoNotCallById);
```

Comment:
ah yes, we can remove this completely. I think we forgot to do that after our conversation with you